Tools/autotest/arducopter.py

```diff
@@ -9976,6 +9976,15 @@ def SMART_RTL_EnterLeave(self):
         self.change_mode('ALT_HOLD')
         self.change_mode('SMART_RTL')
 
+    def SMART_RTL_Repeat(self):
+        '''Test whether Smart RTL catches the repeat'''
+        self.takeoff(alt_min=10, mode='GUIDED')  
+        self.set_rc(3, 1500)
```

Comment:
```suggestion
        self.hover()
```
... that could be useful if you're running a frame other than the default, boring one.

We don't do this in many places yet, and it doesnt' actually do anything but set it to 1500... but aspirationally...